### ISSUE TITLE
Could you please provide a pretrained model?

### ISSUE THREAD

**Issue by dfzsgjshzfj** [HugoCMU/gazeGAN#8]
Hello.
Your work is cool, I really want to experience it.
However, I have some trouble in training a new model, could you please provide a pretrained model?
Thank you.

**Comment by Bhartendu-Kumar** [comment]
Yes, though Author's created codes are inspiring and makes it easy to train a model. Yet, with my resources, I can just train on small dataset. Please provide your model!

**Comment by hu-po** [comment]
Sorry but I am very busy with a different project and I cannot put time into this :( . Best of luck guys!

**Comment by Bhartendu-Kumar** [comment]
No issues, we are already grateful about the stuff that is present. Wishing you luck and may your generosity help you to fulfill your destiny.

**Comment by rafikg** [comment]
hi @Bhartendu-Kumar can you please provide a model if you have already trained the model? I got problem generating synthetic data
